src/CommonITILValidation.php

```diff
@@ -969,201 +969,116 @@ public function showSummary(CommonDBTM $item)
-            echo Dropdown::getValueWithUnit($item->fields["validation_percent"], "%");
-            echo "</td>";
-        }
-        echo "</tr>";
-        echo "</table>";
-        if ($canadd) {
-            Html::closeForm();
-        }
-
         $iterator = $DB->Request([
             'FROM'   => $this->getTable(),
             'WHERE'  => [static::$items_id => $item->getField('id')],
             'ORDER'  => 'submission_date DESC'
         ]);
 
-        $colonnes = ['', _sx('item', 'State'), __s('Request date'), __s('Approval requester'),
-            __s('Request comments'), __s('Approval status'),
-            __s('Requested approver type'), __s('Requested approver'),
+        $columns = ['', _x('item', 'State'), __('Request date'), __('Approval requester'),
+            __('Request comments'), __('Approval date'),
+            __('Requested approver type'), __('Requested approver'), __('Approval Comment'), __('Documents')
         ];
-        $nb_colonnes = count($colonnes);
-
-        echo "<table class='tab_cadre_fixehov'>";
-        echo "<tr class='noHover'><th colspan='" . $nb_colonnes . "'>" . __s('Approvals for the ticket') .
-           "</th></tr>";
-
-        if ($canadd) {
-            /** @var CommonITILObject $item */
-            if (
-                !in_array($item->fields['status'], array_merge(
-                    $item->getSolvedStatusArray(),
-                    $item->getClosedStatusArray()
-                ))
-            ) {
-                echo "<tr class='tab_bg_1 noHover'><td class='center' colspan='" . $nb_colonnes . "'>";
-                echo "<a class='btn btn-outline-secondary' href='javascript:viewAddValidation" . $tID . "$rand();'>";
-                echo __s('Send an approval request') . "</a></td></tr>\n";
-            }
-        }
-        if (count($iterator)) {
-            $header = "<tr>";
-            foreach ($colonnes as $colonne) {
-                $header .= "<th>" . $colonne . "</th>";
+        $nb_columns = count($columns);
+        $transmessage = sprintf(__('%1$s = %2$s'), $item->getTypeName(1), $item->fields["name"]);
+
+        Session::initNavigateListItems(
+            $this->getType(),
+            //TRANS : %1$s is the itemtype name, %2$s is the name of the item (used for headings of a list)
+                                    sprintf(
+                                        __('%1$s = %2$s'),
+                                        $item->getTypeName(1),
+                                        $item->fields["name"]
+                                    )
+        );
+        foreach ($iterator as $row) {
+            $canedit = $this->canEdit($row["id"]);
+            Session::addToNavigateListItems($this->getType(), $row["id"]);
+            $bgcolor = self::getStatusColor($row['status']);
+            $status  = "<div class='badge text-dark fw-normal fs-4 text-wrap' style='background-color:" . $bgcolor . ";'>" . self::getStatus($row['status']) . "</div>";
```

Comment:
Accessibility is better if you add a border with the status color and leave the background as-is. No need to worry about color contrast between the text and background then.